packages/vue-client/src/components/SeatComponent.vue

```diff
@@ -11,12 +11,15 @@ defineProps<{
   player_n: number;
   selected?: number;
   time_control: IPerPlayerTimeControlBase | null;
+  is_players_turn: boolean;
 }>();
 </script>
 
 <template>
   <div
-    v-bind:class="`seat ${selected === player_n ? 'selected' : ''}`"
+    v-bind:class="`seat ${selected === player_n ? 'selected' : ''} ${
+      is_players_turn ? 'to-move' : ''
+    }`"
```

Comment:
This looked a bit verbose to me, so I checked if Vue has a nicer way to do this. And there's this: https://vuejs.org/guide/essentials/class-and-style.html

Reply:
Thanks for the suggestion!
I changed it. Is is okay like this?

Reply:
Yes, that looks nicer, thanks. :slightly_smiling_face: